### ISSUE TITLE
Track “Purchase method” for each item 

### ISSUE THREAD

**Issue by brenletheninja** [sysadminsmedia/homebox#117]
### What is the problem you are trying to solve with this feature?

One of my primary uses of homebox is to track warranties. Some credit cards offer purchase protection and longer warranties, so it is important to know which method/card was used so that I can know whether the item is covered under warranty through the card.

### What is the solution you are proposing?

MVP: add a simple text field for purchase method

North Star: add a purchase method management where you can save purchase methods globally and any associated benefits/warranties. Then a drop down selection within the purchase details of each item. 

Alternative MVP: allow global custom fields so I can add what I need once instead of for each item 

### What alternatives have you considered?

Currently I add a custom field to each item, but it is cumbersome. 

### Additional context

Willing to sponsor and/or help with development though I have limited dev experience. 

### Contributions

- [X] I have searched through existing issues and feature requests to see if my idea has already been proposed.
- [X] If this feature is accepted, I would be willing to help implement and maintain this feature.
- [X] If this feature is accepted, I'm willing to sponsor the development of this feature.

**Comment by katosdev** [comment]
Makes sense to me, and an MVP implementation shouldn’t be hugely difficult, I don’t think.

**Comment by TerraTech** [comment]
Will this also include tracking "Order Number"?  e.g.  When I order things from Amazon, I'd like to easily add the Order # instead of having to add a custom field every time.

**Comment by tankerkiller125** [comment]
Please note we're converting this issue into a discussion as per our new procedures/policies. You can read more about that in our announcement #712
